test/functionalTest/cases/3177_updateContext_APPEND_CPr_Registration/updateContext_APPEND_CPr_Reg.test

```diff
@@ -0,0 +1,212 @@
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+updateContext actionType APPEND with Context Provider Registration
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Register CP.
+#
+echo "01. Register CP."
+echo "========================================================================"
+payload='{
+  "contextRegistrations": [
+    {
+      "entities": [
+        {
+          "type": "Floor",
+          "isPattern": "false",
+          "id": "Floor1"
+        }
+      ],
+      "attributes": [
+        {
+          "name": "temperature",
+          "type": "degree",
+          "isDomain": "false"
+        }
+      ],
+      "providingApplication": "${Context_Provider_URL}"
+    }
+  ],
+  "duration": "PT1H"
+}'
+orionCurl --url "/v1/registry/registerContext" --payload "$payload"
+REG_ID=$(echo "$_response" | grep registrationId | awk -F\" '{ print $4}')
+
+#
+# 02. Creating an entity in CB with actionType APPEND by using NGSIv1 .
+#
+echo "02. Creating an entity in CB with actionType APPEND by using NGSIv1."
+echo "========================================================================"

```

Comment:
Adjust line length to the one above. I mean, remove "=" in L66 so it gets same length that L65

Reply:
Fixed in 589148e